projects/rero/ng-core/src/lib/record/export-button/export-button.component.html

```diff
@@ -0,0 +1,81 @@
+<!--
+  RERO angular core
+  Copyright (C) 2022 RERO
+  Copyright (C) 2022 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="exportOptions && exportOptions.length > 0">
+  <!-- check single export format -->
+  <ng-container *ngIf="exportOptions.length == 1; else  multipleFormats">
+    <!-- check if we can export list -->
+    <ng-container *ngIf="!exportOptions[0]?.disabled; else cannotExport">
+      <a id="search-export-button"
+         role="button" class="ml-2" [class]="cssClass"
```

Comment:
for me `ml-2` and all "display" classes should not be part of the core component.
It should be pass either using "cssClass", either on class of the component selection either using CSS (button + button selector)

Reply:
ok